### PR TITLE
Marketing Asset Guidelines

### PR DESCRIPTION
**Pull request by jasonblais**
@lindy65 please help with initial review on

 - structure of the document
 - asset guidelines
 - style guide and grammar review

Notes: We might want to add specific size guidelines for asset and screenshot dimensions depending on where they appear (Twitter banners, blog posts, ...)

### CODE REVIEW

File: source/process/asset-guidelines.rst [hunk 1]
```diff
@@ -0,0 +1,217 @@
...
+
+  - Text of name should be at least as wide as "for Mattermost"
+  - The space between the bottom of the integration name and "for Mattermost" should be the same distance as in the reference screenshot above
+  - Vertically center the combined image of the integration name and the text "for Mattermost"
+
+Step-by-Step Asset Review Guide for Ops
+----------------------------------------
+
+1. Upload the artists "work in progress" files to the `Archive Dropbox sub-folder <https://www.dropbox.com/home/marketing/Twitter/archive>`_ in `Twitter Marketing <https://www.dropbox.com/sh/13h55hakbvm7iva/AAARooC0rV8JCKBI_8VUj_tga?dl=0>`_
+2. In the `Contributors team <https://pre-release.mattermost.com/core/channels/i18n-japanese>`_, find the appropriate conversation thread for the queued project, and add a comment mentioning @jason with the following:
```
Comment (lindy65) [inline source/process/asset-guidelines.rst]: Did you mean the "Integrations and Apps" channel?

File: source/process/asset-guidelines.rst [hunk 2]
```diff
@@ -0,0 +1,217 @@
...
+
+    Example:
+
+    .. code-block:: none
+
+      @jason Here is [Matterbridge](https://github.com/42wim/matterbridge) from [42wim](https://github.com/42wim), which is a sample bridge between Mattermost, IRC, XMPP, Gitter and Slack.
+      
+      Is this something Mattermost would like to promote?
+
+2. If the proposal is accepted, Ops reaches out to the integration creator by opening a Github issue with the following text:
```
Comment (lindy65) [inline source/process/asset-guidelines.rst]: "Github" > "GitHub"

File: source/process/asset-guidelines.rst [hunk 3]
```diff
@@ -0,0 +1,217 @@
...
+4. Ops replies to each new Wufoo post with a numbered project title and project status. Example:
+
+    .. code-block:: none
+
+      #### 64) Cloud Foundry Integration (work queued)
+
+Media Articles
+^^^^^^^^^^^^^^^
+
+When a relevant article on Mattermost is found in the news, add a new entry into `this Google document <https://docs.google.com/document/d/1kwCmn6JYeORdLV0noIk4iaxZx0iqR6OWUuzw5cZl6rA/edit>`_ and follow the process specified there.
```
Comment (lindy65) [inline source/process/asset-guidelines.rst]: I made a couple of comments in the Google doc as it seems a few links are out of date?

**Comment by jasonblais** [comment]
Thanks @lindy65! Ready for review

**Comment by jasonblais** [comment]
Taking WIP label out, this should be ready for PM review as well

**Comment by esethna** [comment]
Looks good, submitted a small tweak to the relative links

**Pull request merged**